lib/samples/style_graphics_with_symbols/style_graphics_with_symbols.dart

```diff
@@ -0,0 +1,336 @@
+    _createPoints();
+    _createPolygon();
+    _createPolyline();
+    _createText();
+
+    // Update the extent to encompass all of the symbols.
+    _setExtent();
+
+    // Set the ready state variable to true to enable the sample UI.
+    setState(() => _ready = true);
+  }
+
+  void _createPoints() {
+    // Create a red circle simple marker symbol.
+    SimpleMarkerSymbol redCircleSymbol = SimpleMarkerSymbol(
+        style: SimpleMarkerSymbolStyle.circle, color: Colors.red, size: 10,);
+
+    // Create graphics and add them to graphics overlay.
+    Graphic graphic = Graphic(
+      geometry: ArcGISPoint(
+        x: -2.72,
+        y: 56.065,
+        spatialReference: SpatialReference.wgs84,
+      ),
+      symbol: redCircleSymbol,
+    );
+    _graphicsOverlay.graphics.add(graphic);
+
+    graphic = Graphic(
+      geometry: ArcGISPoint(
+        x: -2.69,
+        y: 56.065,
+        spatialReference: SpatialReference.wgs84,
+      ),
+      symbol: redCircleSymbol,
+    );
+    _graphicsOverlay.graphics.add(graphic);
+
+    graphic = Graphic(
+      geometry: ArcGISPoint(
+        x: -2.66,
+        y: 56.065,
+        spatialReference: SpatialReference.wgs84,
+      ),
+      symbol: redCircleSymbol,
+    );
+    _graphicsOverlay.graphics.add(graphic);
+
+    graphic = Graphic(
+      geometry: ArcGISPoint(
+        x: -2.63,
+        y: 56.065,
+        spatialReference: SpatialReference.wgs84,
+      ),
+      symbol: redCircleSymbol,
+    );
+    _graphicsOverlay.graphics.add(graphic);
+  }
+
+  void _createPolyline() async {
+    // Create a purple simple line symbol.
+    SimpleLineSymbol lineSymbol = SimpleLineSymbol(
+      style: SimpleLineSymbolStyle.dash,
+      color: Colors.purple,
+      width: 10,
+    );
+
+    // Build a polyline.
+    final polylineBuilder =
+        PolylineBuilder(spatialReference: SpatialReference.wgs84);
+    polylineBuilder.addPoint(
+      ArcGISPoint(
+        x: -2.715,
+        y: 56.061,
+      ),
+    );
+    polylineBuilder.addPoint(
+      ArcGISPoint(
+        x: -2.6438,
+        y: 56.079,
+      ),
+    );
+    polylineBuilder.addPoint(
+      ArcGISPoint(
+        x: -2.638,
+        y: 56.079,
+      ),
+    );
+    polylineBuilder.addPoint(
+      ArcGISPoint(
+        x: -2.636,
+        y: 56.078,
+      ),
+    );
+    polylineBuilder.addPoint(
+      ArcGISPoint(
+        x: -2.636,
+        y: 56.077,
+      ),
+    );
+    polylineBuilder.addPoint(
+      ArcGISPoint(
+        x: -2.637,
+        y: 56.076,
+      ),
+    );
+    polylineBuilder.addPoint(
+      ArcGISPoint(
+        x: -2.715,
+        y: 56.061,
+      ),
+    );
+
+    final polyline = polylineBuilder.toGeometry();
+
+    // Create the graphic with polyline and symbol.
+    final graphic = Graphic(geometry: polyline, symbol: lineSymbol);
+
+    // Add graphic to the graphics overlay.
+    _graphicsOverlay.graphics.add(graphic);
+  }
+
+  void _createPolygon() async {
```

Comment:
I think there should be a comment here, something like `// Create a green dash line symbol.`